client/signup/accordion-form/form-components.tsx

```diff
@@ -8,14 +8,21 @@ import FormTextInput from 'calypso/components/forms/form-text-input';
 import FormTextArea from 'calypso/components/forms/form-textarea';
 import SocialLogo from 'calypso/components/social-logo';
 
-const Label = styled( FormLabel )`
+// TODO: This should probably moved out to a more difm components section
```

Comment:
Just a naming suggestion because these are not actually not related to the accordion so maybe can live outside in a more generic components folder.

Reply:
Pardon my English :D
```suggestion
// TODO: This probably should be moved out to a more suitable folder name like difm-components